src/htdocs/model/record_model.php

```diff
@@ -127,6 +128,8 @@ public function getLastAvg($deviceId, $hours) {
         foreach (RecordModel::FIELDS as $i => $f) {
             $data[$f] = $row[$i];
         }
+        $data['count'] = intval($row['COUNT(*)']);
```

Comment:
If we provide the `cnt` alias as above, we can use:
```
$row['cnt']
```

Why is the `intval()` required here?

Reply:
If I remember correctly it was a string and I wanted a correct type to be exposed to JavaScript/HTML.
(I think it is not needed now but it can help to avoid some potential mistake in future)

Let me know if you want to change it.

Reply:
In the `foreach` above we don't cast the output (even though it should be a number as well), so if it's not required let's strip it.